frontend/src/modules/projects/views/projectsView.js

```diff
@@ -1,280 +1,182 @@
           }
         },
-        success: _.bind(function(data) {
-          // On successful collection fetching set lazy render to enabled
-           if (this.collectionLength === this.collection.length) {
-              this.shouldStopFetches = true;
-          } else {
-              this.shouldStopFetches = false;
-              this.collectionLength = this.collection.length;
-          }
+        success: _.bind(function(collection, response) {
           this.isCollectionFetching = false;
+          this.fetchCount += response.length;
+          // stop further fetching if this is the last page
+          if(response.length < this.pageSize) this.shouldStopFetches = true;
+
+          this.$('.no-projects').toggleClass('display-none', this.fetchCount > 0);
+          if(typeof cb === 'function') cb(collection);
         }, this)
       });
     },
 
-    appendProjectItem: function(projectModel) {
-      projectModel.attributes.title=this.highlight(projectModel.attributes.title)
-
-      if (!projectModel.isEditable()) {
-        this.getProjectsContainer().append(new SharedProjectView({ model: projectModel }).$el);
-      } else {
-        this.getProjectsContainer().append(new ProjectView({ model: projectModel }).$el);
+    doLazyScroll: function(e) {
+      if(this.isCollectionFetching) {
+        return;
       }
+      var $el = $(e.currentTarget);
+      var pxRemaining = this.getProjectsContainer().height() - ($el.scrollTop() + $el.height());
+      // we're at the bottom, fetch more
+      if (pxRemaining <= 0) this.fetchCollection();
     },
 
-    highlight: function(text) {
-      var search  = this.getUserPreferences().search || '';
-      // replace special characters: .*+?|()[]{}\$^
-      search.replace(/[.*+?|()\[\]{}\\$^]/g, "\\$&");
-      // add the span
-      return text.replace(new RegExp(search, "gi"), function(term) {
-        return '<span class="highlighted">' + term + '</span>';
-      });
-    },
-
-    addTag: function(filterType) {
-      // add filter to this.filters
-      this.tags.push(filterType);
-      this.filterCollection();
-    },
-
-    removeTag: function(filterType) {
-      // remove filter from this.filters
-      this.tags = _.filter(this.tags, function(item) { return item != filterType; });
-      this.filterCollection();
-    },
-
-    filterCollection: function() {
-      this.search.tags = this.tags.length
-          ? { $all: this.tags }
-          : null ;
-      this.updateCollection(true);
-    },
-
-    convertFilterTextToPattern: function(filterText) {
-      var pattern = '.*' + filterText.toLowerCase() + '.*';
-      return { title: pattern};
+    doLayout: function(layout) {
+      var layouts = ["grid", "list"];
+      if(_.indexOf(layouts, layout) === -1) {
+        return;
+      }
+      var classSuffix = '-layout';
+      this.getProjectsContainer()
+        .removeClass(layouts.join(classSuffix + ' ') + classSuffix)
+        .addClass(layout + '-layout');
```

Comment:
Not crucial to this PR but would you be privvy to using a data attribute for this instead? Could be cleaner / more flexible.

For example:
```js
this.$el.attr('data-layout', layout);
```
Earlier on in the file we would set the default:
```js
attributes: {
  'data-layout': 'grid'
}
```

[`projects.less`](https://github.com/adaptlearning/adapt_authoring/blob/issue/1759/frontend/src/modules/projects/less/projects.less#L41-L114) and [`projects.hbs`](https://github.com/adaptlearning/adapt_authoring/blob/issue/1759/frontend/src/modules/projects/templates/projects.hbs#L5) would need tweaks too.